### PR TITLE
Update blob auditing spec from swagger generator

### PR DESCRIPTION
**Pull request by jaredmoo**
Used SQL swagger generator to regenerate the blob auditing spec. The path is represented in a different way (blob auditing policy parameter with single enum value, instead of hardcoded in path) but otherwise there should be no functional difference.

-----

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by jaredmoo** [comment]
@yaakoviyun , @ayeletshpigelman , @nathannfan FYI

File: arm-sql/2015-05-01-preview/swagger/blobAuditingPolicies.json [hunk 1]
```diff
@@ -106,120 +132,40 @@
...
           "description": "Specifies the state of the policy. If state is Enabled, storageEndpoint and storageAccountAccessKey are required.",
-          "type": "string",
           "enum": [
             "Enabled",
             "Disabled"
           ],
-          "x-ms-enum":{
+          "type": "string",
+          "x-ms-enum": {
             "name": "BlobAuditingPolicyState"
```
Comment (nathannfan) [inline arm-sql/2015-05-01-preview/swagger/blobAuditingPolicies.json]: Not a big deal, but are you on the latest?  Generator should be adding "modelAsString": "false" here

Comment (jaredmoo) [inline arm-sql/2015-05-01-preview/swagger/blobAuditingPolicies.json]: No, this is from yesterday :) I'll fix it

**Comment by jaredmoo** [comment]
Validations to fix:
1. SecurityDefinitionsStructureValidation
2. All the examples validations

The rest are noise

**Comment by balajikris** [comment]
Thanks @jaredmoo . I'll take a look.

**Comment by jaredmoo** [comment]
All the remaining warnings are noise. Ready to review now.

**Review by nathannfan — APPROVED** [review]

**Comment by balajikris** [comment]
Hey @jaredmoo -

Regarding this error:
~~~
{
  "type": "Error",
  "code": "OperationsAPIImplementationValidation",
  "message": "Operations API must be implemented for '/providers//operations'.",
  "id": "M3023",
  "validationCategory": "RPCViolation",
  "providerNamespace": null,
  "resourceType": null,
  "sources": [
    "file:///e:/repos/azure-rest-api-specs/arm-sql/compositeSql.json:1:0 ($)"
  ]
}
~~~

are you saying this is a red-herring _or_ it doesn't apply to your case _or_ you don't want to fix it now ?

**Comment by jaredmoo** [comment]
Operations for sql is implemented in a different file - https://github.com/Azure/azure-rest-api-specs/blob/master/arm-sql/2014-04-01/swagger/operations.json

**Comment by jaredmoo** [comment]
So... red herring I guess? :) We have a 2015-05-01-preview version of /operations coming online very soon, and we plan to consolidate all 2015-01-01-preview version apis including operations into a single swagger file which will fix this warning.

**Comment by balajikris** [comment]
Thanks @jaredmoo . Starting an internal thread to investigate this potential false error. I'm okay with the changes, will sign-off.

**Review by balajikris — APPROVED** [review]

**Pull request merged**